UnitySamples/FBLeaderboard/Docs/FBLeaderboardTutorial.md

```diff
@@ -0,0 +1,301 @@
+
+It may be the case (post re-install or on a different device for example) that the player logs in to ChilliConnect with a FB token and already has an account. The player may have even made progress in the game forgetting that they had an existing account. Different games handle this in different ways but the most common solution is to determine whether the local anonymous account or the FB linked account has more progress and use that data.
+
+## Leaderboard
+
+Compared with the login flow the actual leaderboard aspect of the demo is incredibly straightforward. The source for this section is found in 'LeaderboardSystem.cs' and comprises two main parts: posting scores and retrieving scores.
+
+### Submitting scores to leaderboard
+
+Use the leaderboard API to post a score to a named leaderboard (in our case the leaderboard key is "HIGHSCORES").
+
+```c#
+public void PostScore(int score)
+{
+    string LEADERBOARD_KEY = "HIGHSCORES";
+    m_chilliConnect.Leaderboards.AddScore(new AddScoreRequestDesc(score, LEADERBOARD_KEY), (request, response) => OnScoreAdded(response), (request, error) => Debug.LogError(error.ErrorDescription));
+}
+```
+The response contains the players score and rank on the leaderboard.
+
+```c#
+private void OnScoreAdded(AddScoreResponse response)
+{
+    Debug.Log(string.Format("Player's high score {0}. Player is {1}/{2} on global leaderboard", response.Score, response.GlobalRank, response.GlobalTotal));
+}
+```
+
+NOTE: The score is the player's score on the leaderboard which may not be the score they just submitted (in our case it returns their highscore).
+
+### Fetching leaderboard scores
+
+Fetching scores is also a straightforward process (again it requires the leaderboard key):
+
+```c#
+public void FetchFriendLeaderboard()
+{
+    var desc = new GetScoresForFacebookFriendsRequestDesc(LEADERBOARD_KEY);
+
+    //Setting this to true ensures the logged in player appears in the leaderboard results
+    desc.IncludeMe = true;
+
+    m_chilliConnect.Leaderboards.GetScoresForFacebookFriends(desc, (request, response) => OnLeaderboardFetched(response), (request, error) => Debug.LogError(error.ErrorDescription));
+}
+
+```
+
+Our demo is only interested in showing scores for a player's FB friends but ChilliConnect provides other methods for querying the leaderboard:
+
+- 'GetScoresForChilliConnectIds': Returns leaderboard scores for the given players (if they have submitted a score).
+- 'GetScores': Used to display the entire leaderboard (NOTE: the API returns leaderboard pages and the page must be specified in the request).
+- 'GetScoresAroundPlayer': Returns scores above and below the player in the leaderboard.
+
+The response contains a collection of 'FacebookScore' objects, each object has a friend's facebook name, profile picture url, score and position on the leaderboard:
+
+```c#
+private void OnLeaderboardFetched(GetScoresForFacebookFriendsResponse response)
+{
+    PopulateLeaderboard(response.Scores);
```

Comment:
This is named differently in the code `OnLeaderboardRefreshed(response.Scores.ToArray());`